docs/advanced-topics/wes/wes-cwl-agc-tutorial.rst

```diff
@@ -0,0 +1,247 @@
+WES Tutorial for running CWL workflows through the Toil Engine on AWS AGC
+=========================================================================
+
+.. note::
+    Although the process for running WDL workflows through the Cromwell engine on AWS AGC is very similar to the process
+    described in this tutorial. There are a few differences. So, for information on how to launch a WDL workflow on AWS
+    AGC please consult this :doc:`tutorial <wes-wdl-agc-tutorial>`.
+
+Amazon Web Services' (AWS) Amazon Genomics CLI (AGC) is a command line tool for launching cloud infrastructure
+within AWS accounts that can be used to execute genomics workflows. The infrastructure deployed by AGC implements the WES
+standard, and thus can be directly communicated with by using the Dockstore CLI.
+
+Check out the official AGC `GitHub page <https://github.com/aws/amazon-genomics-cli>`_.
+
+Download and Install AWS AGC
+----------------------------
+AGC provides a `quick-start guide <https://aws.github.io/amazon-genomics-cli/docs/getting-started/>`_ for initial setup
+and getting familiar with the tool. The following workflow execution tutorial will cover all steps for using AGC once
+it has been `installed <https://aws.github.io/amazon-genomics-cli/docs/getting-started/installation/>`_.
+
+Tutorial Topics
+----------------
+The following CWL workflow tutorials will cover:
+
+1. Deploying an AGC project and context
+2. Configuring the Dockstore CLI to communicate with AGC infrastructure
+3. Launching a workflow
+
+
+Configuring AGC and the Dockstore CLI
+----------------------------------------
+1. Create a file named ``agc-project.yaml`` that contains:
+
+    .. code:: text
+
+        name: dockstoreAgcTutorialProject
+        schemaVersion: 1
+        contexts:
+          ctx2:
+            engines:
+              - type: cwl
+                engine: toil
+
+This will create an AGC project named ``dockstoreAgcTutorialProject``, with a single context named ``ctx2``.
+
+.. note::
+
+    For AGC infrastructure to interact with an S3 resource, the desired S3 bucket must be specified in the project's ``agc-project.yaml`` file
+    and your AWS account must already have access to the S3 resource. For more information on how to this, please click `here <https://aws.github.io/amazon-genomics-cli/docs/concepts/data/>`_.
+
+
+2. Activate AGC on your account. If this is your first time running AGC on an account, this may take a few minutes.
+
+    .. code:: text
+
+        agc account activate
+
+3. Deploy an AGC context by running the below command in the same directory as ``agc-project.yaml``. This may take a few minutes.
```

Comment:
I forget, doesn't this step, or maybe it's the previous one, take around 20 minutes (more than a few)? I may be misremembering...

Reply:
Changed it to stating that it takes 10 minutes to run.

Also added info on how to destroy a context, that is what takes 20 minutes.